src/components/Order.js

```diff
@@ -0,0 +1,58 @@
+import React, {useEffect, useState} from 'react';
+import {useParams} from 'react-router-dom';
+import {fetchOrderById} from '../api';
+
+
+const Order = (props) => {
+    const {token} = props;
+    const { orderId } = useParams();
```

Comment:
Optional - Redirect (`history.push()` to a different url (home?) if the `order.userId` does not match state `user.id` (passed as prop?) OR if the state.user.isAdmin is true? (but the second part might come later?)

Reply:
Fixed